src/elements/overlay/overlay.visual.spec.ts

```diff
@@ -0,0 +1,60 @@
+import { html, type TemplateResult } from 'lit';
+
+import { describeViewports, visualDiffDefault } from '../core/testing/private.js';
+
+import './overlay.js';
+import type { SbbOverlayElement } from './overlay.js';
+
+describe(`sbb-overlay`, () => {
+  const defaultArgs = {
+    expanded: false,
+    backButton: false,
+    negative: false,
+  };
+
+  const template = ({ expanded, backButton, negative }: typeof defaultArgs): TemplateResult => html`
+    <sbb-overlay ?negative=${negative} ?expanded=${expanded} ?back-button=${backButton}>
+      <div class="overlay-content">
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
+        labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco
+        laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in
+        voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat
+        cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+      </div>
+    </sbb-overlay>
+  `;
+
+  const openOverlay = (setup: any): void => {
```

Comment:
Could it be typed instead of any? :/

Reply:
No, but we can consider to export the VisualDiffSetupBuilder class

Reply:
@kyubisation What do you think? I remember you preferred to have it not exported.

Reply:
Yes, but purely due to not having a necessity. If it helps in this case, then we can export it.

Reply:
ok, thanks! So I'd prefer exporting it over having `any` type